oshi-core/src/main/java/oshi/software/os/linux/LinuxOperatingSystem.java

```diff
@@ -262,6 +262,15 @@ public OSProcess getProcess(int pid) {
         proc.setBytesRead(ParseUtil.parseLongOrDefault(MapUtil.getOrDefault(io, "read_bytes", ""), 0L));
         proc.setBytesWritten(ParseUtil.parseLongOrDefault(MapUtil.getOrDefault(io, "write_bytes", ""), 0L));
         
+        //gets the open files count
+        try {
+            String openFilesString = ExecutingCommand.getFirstAnswer(String.format("ls -f /proc/%d/exe | wc -l", pid));
+            if (openFilesString!=null)
+                proc.setOpenFiles(Long.parseLong(openFilesString));

```

Comment:
Please use the Util's `parseLongOrDefault()` method to catch any errors here (and elsewhere, won't repeat this comment).  I believe that will avoid having to catch exceptions at this level.